rmf_site_editor/src/site/model.rs

```diff
@@ -783,25 +786,42 @@ pub fn update_model_instances<T: Component + Default + Clone>(
     }
 }
 
+/// Unique UUID to identify issue of duplicated lift names
```

Comment:
Nit, update the comment after copying the snippet

Reply:
[a6d4139](https://github.com/open-rmf/rmf_site/pull/260/commits/a6d4139219d7eabae316a7946928e954c0163db0)